.github/workflows/build-and-test.yml

```diff
@@ -14,14 +14,14 @@ jobs:
     timeout-minutes: 20
 
     steps:
-    - name: Checkout Swift-WebBrowser
-      uses: actions/checkout@v4.1.1
+    - name: Checkout repo
+      uses: actions/checkout@v4.2.2
 
     - name: Setup Visual Studio Development Environment
-      uses: compnerd/gha-setup-vsdevenv@main
+      uses: compnerd/gha-setup-vsdevenv@f1ba60d553a3216ce1b89abe0201213536bc7557 # main as of 2024-11-12
```

Comment:
Can we get compnerd to do some releases?

Reply:
What does @compnerd think?